src/common/Components/StarRating/index.unit.test.js

```diff
@@ -0,0 +1,10 @@
+import { mount } from 'enzyme';
+
+import StarRating from '.';
+
+describe('StarRating', () => {
+  it('Renders the correct amount of stars', () => {
+    const component = mount(<StarRating filledStars={4} halfFilledStars={1} />);
```

Comment:
This should be under the `describe`. Usually the `it` should only have the expect (sometimes there's something else needed), and the `describe` sets all up for the test itself